### PR TITLE
2021 Annual Report: SIG Docs

### PR DESCRIPTION
**Pull request by divya-mohan0209**
Signed off by: Divya Mohan divya.mohan0209@gmail.com

Which issue(s) this PR fixes:

Issue: #6371

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
/assign
@jimangel @reylejano @natalisucks

**Comment by divya-mohan0209** [comment]
cc: @parispittman @jimangel @reylejano @natalisucks

File: sig-docs/annual-report-2021.md
Comment (sftim) [inline sig-docs/annual-report-2021.md:72]: Does this make sense for SIG Docs?

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md]: Addressed in the latest commit.

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:7]: link to the shadow program docs

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:8]: include a link to the mailing list entries with Jim asking for help and then the announcement email

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:12]: add any and all applicable links here - issues and proposal doc etc

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:13]: same here - any links that you can point folks too, even if you have to create the issue now as a placeholder that would be fine. do you want to shoutout anyone doing this work?

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:10]: would another one be the consistent maintenance of the docset and strategies therein on how to make that happen? what about some other initiatives that I see on your agendas? do you have project board for the group or any subprojects?

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:31]: new contributors? members? reviewers? all?

Comment (parispittman) [reply sig-docs/annual-report-2021.md:31]: PR wranglers? 
get really detailed here and include links to any membership documentation, etc

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:37]: do y'all do anything with this data? example: make sure these pages are always updated and fresh?

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:5]: any milestones hit or really amazing metrics to share about how much work y'all accomplished? example: page views, PRs reviewed, etc?

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:43]: this is good info: We aim to have < 100 open PRs for the English localization, and will take steps as needed if we see the figure climbing much above that.

File: sig-docs/annual-report-2021.md
Comment (parispittman) [reply sig-docs/annual-report-2021.md:5]: did you work with other groups on thorny problems? examples of good collaboration, etc?

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:48]:
```suggestion
   - [It's updated](https://kubernetes.io/docs/contribute/) to the best of our knowledge.
```

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:53]:
```suggestion
   - We have a [well-documented guide](https://kubernetes.io/docs/contribute/) that details how folks can get started and the various ways in which they can scale the contributor ladder.
```

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:54]: have any links to those sessions?

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:63]: is there a statement we can make in this section about the importance of localization and why end users should care?

File: sig-docs/annual-report-2021.md
Comment (parispittman) [reply sig-docs/annual-report-2021.md:31]: did y'all run dim's maintainer audit tooling?

**Review by parispittman — CHANGES_REQUESTED** [review]

File: sig-docs/annual-report-2021.md
Comment (parispittman) [inline sig-docs/annual-report-2021.md:58]: I would call out the companies that do the top 50% and also make a statement about the independents in your group - that is a huge number and speaks to the types of support that your contributor base gets when they contribute

**Review by parispittman — CHANGES_REQUESTED** [review]

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:54]: We didn't have any this past year. I combed through Youtube but only found Jim's, Zach's and Seokho's presentations at previous KubeCons. Would it be worthwhile listing them here? Jim's presentation is already listed below.

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:63]: Yes, I will draft something.

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:10]: We don't have a project board, to the best of my knowledge. Ccing the other leads/chairs on here so that they can provide further insight.
@jimangel @sftim @onlydole @kbhawkey @reylejano @a-mccarthy @natalisucks @tengqm

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:7]: We don't have a doc per se, Paris. @bradtopol please correct me if I'm mistaken here. We announced it during one of our meetings and asked them to reach out to Brad, if interested. The PR Wranglers list then gets updated with the shadows

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:72]: Tbh, I don't think. But @parispittman would you please be able to comment if this is actually relevant for non-code owning SIGs?

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:12]: Cc: @natalisucks @a-mccarthy @bradtopol since I know y'all have been working towards this.

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:13]: Will create the placeholder issue & link here.

File: sig-docs/annual-report-2021.md
Comment (reylejano) [reply sig-docs/annual-report-2021.md:10]: There is a project board for the blog subproject
https://github.com/kubernetes/website/projects/11

File: sig-docs/annual-report-2021.md
Comment (natalisucks) [reply sig-docs/annual-report-2021.md:7]: We can link to the PR Wrangling page itself that has the shadow listing next to each wrangler for the week

Comment (sftim) [reply sig-docs/annual-report-2021.md:7]: Is that kind of detail right for an annual report? It'd be better to have a doc about the shadow program generally but right now we don't.

I don't think it's obvious from the PR wrangler list how you become a shadow, which I think is the key thing Paris would want readers to be able to discover.

(instead of adding a link: how about we open and prioritize an issue about adding those contributor docs)?

File: sig-docs/annual-report-2021.md
Comment (sftim) [reply sig-docs/annual-report-2021.md:31]: maintainer audit tooling? (I don't think I know what that is)

File: sig-docs/annual-report-2021.md
Comment (sftim) [reply sig-docs/annual-report-2021.md:37]: I don't think we do anything with that. I might be wrong though.

File: sig-docs/annual-report-2021.md
Comment (sftim) [reply sig-docs/annual-report-2021.md:54]: There's also https://www.youtube.com/watch?v=pprMgmNzDcw and https://www.youtube.com/watch?v=vz_5EdhHrmI

File: sig-docs/annual-report-2021.md
Comment (sftim) [inline sig-docs/annual-report-2021.md:29]: I'd call out the blog subproject as short on resources / attention. A lot of the publication that's happening is depending on a single editor (me), and the other editors are perhaps even more stretched with other Kubernetes contributions and SIG involvement(s).

File: sig-docs/annual-report-2021.md
Comment (natalisucks) [reply sig-docs/annual-report-2021.md:12]: Issue created here: https://github.com/kubernetes/website/issues/31955

File: sig-docs/annual-report-2021.md
Comment (natalisucks) [reply sig-docs/annual-report-2021.md:7]: Issue created here: https://github.com/kubernetes/website/issues/31956

Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md]: Added issue mentioned by @natalisucks , hence resolving comment

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md]: Added issue mentioned by @natalisucks, hence resolving comment

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md]: Created issue and linked in latest commit.

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md]: Added only the first one, since the second one is very generic and talks about docs in general.

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md]: Added in last commit

File: sig-docs/annual-report-2021.md
Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:5]: I'll be highlighting the ongoing efforts for Dockershim that we flagged off last year in December.

Comment (divya-mohan0209) [reply sig-docs/annual-report-2021.md:5]: cc: @a-mccarthy for the page views/PRs reviewed bit, if you could please pull in the numbers?

File: sig-docs/annual-report-2021.md
Comment (parispittman) [reply sig-docs/annual-report-2021.md:34]: +1 to @sftim - I see the update, too. can you add that exactly detail that Tim mentioned above - one editor and looking for more. this is very important if folks want to continue to have a good SLA for their blog posts.

**Comment by parispittman** [comment]
@sftim @divya-mohan0209 I would add more detail around the blog needs as its general right now and doesn't call into the gravity of the situation with only one editor, etc. after that (or during/now), do a final call for comments from sig-docs members on your mailing list with your normal consensus period (72 hours?) and then we merge!
thank you so much!!!

File: sig-docs/annual-report-2021.md
Comment (sftim) [inline sig-docs/annual-report-2021.md:36]:
```suggestion
   - The blog subproject is particularly short on resources and attention. At the moment a very
     small pool of active editors are the constraint / most critical resource for article publication. One editor
     is involved in the majority of published articles;  other editors are perhaps even more stretched with other
     Kubernetes contributions and involvement with other SIGs.
     - Whilst there is work under way to move other people along the contribution ladder, the blog subproject
       remains constrained.
   - SIG Docs would welcome more contributors to form a stable pool for the blog subproject in particular and
     for SIG Docs, generally.
```

File: sig-docs/annual-report-2021.md
Comment (sftim) [inline sig-docs/annual-report-2021.md:38]: I hope this is written from a sufficiently neutral point of view
```suggestion
    - One localization (Ukrainian) is primarily worked on by people based in Ukraine,
      where the ongoing and intensifying conflict creates challenges that take priority over
      open source contribution.
```

I used “conflict” rather than “invasion” because I imagine that the CNCF would rather take a neutral stance on the Russian invasion.

**Comment by divya-mohan0209** [comment]
@parispittman @sftim : Are we happy for the report to be reviewed by the community? If yes, I shall post the link on the slack channel and amplify it in our meeting tomorrow.

**Comment by sftim** [comment]
No objection from me @divya-mohan0209

File: sig-docs/annual-report-2021.md
Comment (sftim) [inline sig-docs/annual-report-2021.md:47]: Oops, my bad formatting needs a fixup
```suggestion
   - One localization (Ukrainian) is primarily worked on by people based in Ukraine,
     where the ongoing and intensifying conflict creates challenges that take priority over
     open source contribution.
```

**Comment by divya-mohan0209** [comment]
I've merged your suggestions, @sftim . If there are no outstanding suggestions @parispittman , I'd like to put it out on Friday, March 11th and wait for community consensus over our mailing list. Would that be okay?

**Comment by divya-mohan0209** [comment]
Since there were no changes suggested from the community, requesting an lgtm from one of the leads/Paris for merging the PR.

**Comment by reylejano** [comment]
Thank you @divya-mohan0209 
/lgtm

**Pull request merged**
